buildSrc/private/src/main/kotlin/androidx/build/AndroidXPlaygroundRootImplPlugin.kt

```diff
@@ -174,21 +174,30 @@ class AndroidXPlaygroundRootImplPlugin : Plugin<Project> {
     }
 
     private fun RepositoryHandler.addPlaygroundRepositories() {
-        repos.all.forEach { playgroundRepository ->
+        fun RepositoryHandler.addCustomRepo(customRepository: PlaygroundRepository) {
```

Comment:
Odd naming in terms of customRepo vs PlaygroundRepo, maybe lets just align the names?

Reply:
Updated